### DISCUSSION TITLE
[Feature request] Provide support for Oracle as backend database for grafana.

### DISCUSSION THREAD

**Discussion by niranjanpai** [grafana/grafana#51481] [Platform]
Grafana currently supports mysql, sqllite and postgres as backend databases. Can we add support for Oracle also as a backend database? We should be able to store user details, dashboards and sessions in Oracle.

- What Grafana version are you using?
   4.2

**Comment by Necrontyr** [comment]
This feature would be much appreciated in a production setting where Oracle is already heavily used and distributed, so all of our perf metrics live there already , I just want the power of grafana to render them

**Comment by MarkRx** [comment]
+1. Oracle is a common Enterprise database

**Comment by JekRock** [comment]
That would be great!

**Comment by bedrin** [comment]
Looks like Oracle is now supported by xorm. Will it unblock the development?

**Comment by dcamero** [comment]
+1 Oracle would be great for existing enterprise solutions

**Comment by zoltanbedi** [comment]
Oracle is one of our enterprise plugin https://grafana.com/grafana/plugins/grafana-oracle-datasource

**Comment by zackman0010** [comment]
@zoltanbedi Sorry for commenting on a multi-year closed post, but this feature request wasn't completed.

The request was the ability to utilize an Oracle database for [High Availability](https://grafana.com/docs/grafana/next/setup-grafana/set-up-for-high-availability/) purposes, as currently only Postgres and MySQL are supported for HA. To my knowledge, the enterprise plugin only adds the ability to use Oracle as a datasource for dashboards, but doesn't actually give the ability to use it for HA. If I'm wrong, however, please correct me!

**Comment by zoltanbedi** [comment]
@zackman0010 Sorry I misunderstood this feature request. I'm going to convert it to discussion as we use GitHub discussions for feature requests.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82951. Feel free to continue the discussion around this there. Thank you!
